physical-spaces-code-of-conduct.md

```diff
@@ -47,6 +47,7 @@ jokes are not appropriate for any event.
 Harassment comes in many forms, including but not limited to:
   - Offensive comments related to gender, sexual orientation, age, disability,
     physical appearance, body size, race, veteran status, or religion
+  - Inappropriate comments related to a person's usage of a face mask
```

Comment:
```suggestion
  - Inappropriate comments related to a person's use or lack of use of a face mask
```
Mainly thinking about how we don't want anyone to harass people about whatever decision they make, but we could go either way.